tools/defaults.bzl

```diff
@@ -1,9 +1,27 @@
 """Re-export of some bazel rules with repository-wide defaults."""
 load("@build_bazel_rules_typescript//:defs.bzl", _ts_library = "ts_library")
+load("@build_bazel_rules_nodejs//:defs.bzl", _jasmine_node_test = "jasmine_node_test")
 
 def ts_library(tsconfig = None, node_modules = None, **kwargs):
   if not tsconfig:
     tsconfig = "//:tsconfig.json"
   if not node_modules:
     node_modules = "@ngrx_compiletime_deps//:node_modules"
   _ts_library(tsconfig = tsconfig, node_modules = node_modules, **kwargs)
+
+def ts_test_library(node_modules = None, **kwargs):
```

Comment:
Is this an OK pattern of defining my own rules that just set defaults for existing rules similar to what you've done with `ts_library`? Or should I be more explicit in `BUILD` files?

Reply:
This is a macro (you can tell because you didn't call the `rule` factory function) and macros can be the gateway to a bad time. In Google, there are a lot of macros that produce surprising combinations of targets, and make the BUILD files look very different from the result of `bazel query --output=build`.
This usage is okay though, matching the criteria:
- it should only wrap one rule, not multiple
- it should just set defaults

This looks clever and reasonable to me :)